expressEngine/login/utils/login.js

```diff
@@ -21,7 +21,7 @@ class LoginHelper {
 
   createToken = async ({ data }) => {
     try {
-      const { key = 'successive' } = process.env;
+      const { KEY: key = 'successive' } = process.env;
```

Comment:
what is the need to write {KEY: key = 'successive'}, you can write only { KEY = 'successive' }

Reply:
because it is a good approach when we destructure the objects in this way.